entries/a/awin.com.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Awin": {
+    "domain": "awin.com",
+    "tfa": [
+      "totp"
+    ],
```

Comment:
```suggestion
    ],
    "documentation": "https://success.awin.com/s/article/How-to-activate-and-deactivate-the-Two-Step-Verification",
```

Reply:
@Poukame You marked this as resolved without a commit - was that intentional?

Reply:
Sorry, my mistake.